org.archimatetool.Archi.yml

```diff
@@ -0,0 +1,43 @@
+# Only x86 binaries are provided by Archimate
+declare: &url https://www.archimatetool.com/downloads/archi.php?/5.0.2/Archi-Linux64-5.0.2.tgz
+declare: &sha256 de86a688d5c94ec8ec1e77b5ed045845546cc4b6ae25fa39aa8424cdfa3bb586
```

Comment:
Items 2 and 3 are why I think the build is failing. If you put the values in the source module it will work.